cmds/pe/pe.go

```diff
@@ -0,0 +1,51 @@
+// Copyright 2017 the u-root Authors. All rights reserved
+// Use of this source code is governed by a BSD-style
+// license that can be found in the LICENSE file.
+
+// Dump the headers of a PE file.
+//
+// Synopsis:
+//     pe [FILENAME]
+//
+// Description:
+//     Windows and EFI executables are in the portable executable (PE) format.
+//     This command prints the headers in a JSON format.
+package main
+
+import (
+	"debug/pe"
+	"encoding/json"
+	"errors"
+	"flag"
+	"fmt"
+	"log"
+	"os"
+)
+
+func openPE() (*pe.File, error) {
+	switch flag.NArg() {
```

Comment:
Generally not nice to parse flags or use the flag package at all outside of main. You could honestly just do this in main.